packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -961,24 +968,20 @@ export default class Meeting extends StatelessWebexPlugin {
      */
     this.inMeetingActions = new InMeetingActions();
     /**
-     * This is deprecated, please use shareStatus instead.
      * @instance
-     * @type {Boolean}
+     * @type {string}
      * @readonly
      * @public
      * @memberof Meeting
-     * @deprecated after v1.118.13
      */
-    this.isSharing = false;
+    this.shareStatus = SHARE_STATUS.NO_SHARE;
     /**
      * @instance
-     * @type {string}
-     * @readonly
+     * @type {ScreenShareFloorStatus}
      * @public
```

Comment:
I'm not sure if `screenShareFloorState` needs to be public, I know that previous isSharing was public, but for now let's keep it private and we'll only make it public when we find it's required

Reply:
changed to private